draft-ietf-trans-rfc6962-bis.md

```diff
@@ -914,10 +918,9 @@ log's parameters.
     } TimestampedCertificateEntryDataV2;
 ~~~~~~~~~~~
 
-`timestamp` is the NTP Time [RFC5905] at which the certificate or precertificate
-was accepted by the log, measured in milliseconds since the epoch (January 1,
-1970, 00:00 UTC), ignoring leap seconds. Note that the leaves of a log's Merkle
-Tree are not required to be in strict chronological order.
+`timestamp` is unsigned Unix Time [UNIXTIME] at which the certificate or precertificate
```

Comment:
Unix Time is 32-bit and measured in seconds, whereas we're using 64-bit values that are measured in milliseconds.  I think it makes sense to reference the POSIX document though, because we're using the same Epoch, the same byte order, and we treat leap seconds the same way.

My proposed text:
"timestamp" is the date and time at which the certificate or precertificate was accepted by the log, in the form of a 64-bit unsigned number of milliseconds elapsed since the Unix Epoch (1 January 1970 00:00:00 UTC - see [UNIXTIME]), ignoring leap seconds, in network byte order. Note that the leaves of a log's Merkle Tree are not required to be in strict chronological order.

Reply:
Thanks for highlighting that, accepted your text.
I'm wondering if "accepted by the log" is ambiguous and would be subject to language lawyering. I think not (because there's already a marked difference between acceptance of a submission and incorporation of said submission into the tree).

Reply:
I agree that it's not ambiguous.